### PR TITLE
WIP: Fix bad tripcubes created

### LINKED ISSUE #174: convertmc tripcube produces negative slice distances
1. slice distance seems to be filled with a bogus value
2. also "Anonymous" is spelled wrong.
3. add pymchelper and PyTRiP version number to creation info, and check spelling "created"
4. use proper case of PyTRiP98
5. pixel size is a factor 10 wrong
```
$ convertmc -V
0.6.1
$ convertmc tripcube myvox02.bdo
$ cat myvox02.hed 
version 2.0
modality CT
created_by pytrip
creation_info create by pytrip
primary_view transversal
data_type integer
num_bytes 2
byte_order vms
patient_name Anonymeous
slice_dimension 102
pixel_size 0.121093802154
slice_distance -0.303921699524
slice_number 102
xoffset 0
dimx 256
yoffset 0
dimy 256
zoffset 0
dimz 102
z_table no
```

### PR DESCRIPTION
**Pull request by nbassler**
fixes #174 
- [x] slice distance seems to be filled with a bogus value
- [x] also "Anonymous" is spelled wrong.
- [x] add pymchelper and PyTRiP version number to creation info, and check spelling "created"
- [x] use proper case of PyTRiP98
-  [x] pixel size is a factor 10 wrong

### CODE REVIEW

**Comment by nbassler** [comment]
@grzanka @ant6 any hint on how do I properly access pytrip and pymchelper version names from writer/trip98.py?

File: pymchelper/writers/trip98.py [hunk 1]
```diff
@@ -1,8 +1,9 @@
 import time
 import logging
 import os
-
 import numpy as np
+from pymchelper import __version__ as _pmcversion
+from pytrip import __version__ as _ptversion
```
Comment (nbassler) [inline pymchelper/writers/trip98.py]: help wanted

Comment (grzanka) [inline pymchelper/writers/trip98.py]: It looks correct - does it work ?

Comment (grzanka) [inline pymchelper/writers/trip98.py]: The problem is that on purpose `pytrip` is not on the list of requirements for pymchelper.
Support of pytrip related functions is for "advanced users" who will install pytrip by themselves.
It was done like this for two reasons:
 - most of pymchelper users work with simple Fluka/SH12A simulation and don't need such heavy dependency
 - installation of pytrip under Windows and old Linuxes is problematic - it depends on many packages (scipy/paramiko etc)

I'd reccommend to do it in following way:
 - make an pytrip import and if it fails, then write a message to the user: "please install pytrip with `pip install pytrip98`"
 - if it succeeds then set the version to correct one
 - if it fails, then stop further execution

Comment (grzanka) [inline pymchelper/writers/trip98.py]: Another thing - move pytrip import down to the method which needs pytrip version and access to its API (i.e. line 16)

**Comment by grzanka** [comment]
I see a merge conflict, probably you need to rebase.

**Pull request merged**